pandas/tests/scalar/timestamp/test_timezones.py

```diff
@@ -141,9 +124,9 @@ def test_tz_localize_ambiguous_raise(self):
         with pytest.raises(AmbiguousTimeError, match=msg):
             ts.tz_localize("US/Pacific", ambiguous="raise")
 
-    def test_tz_localize_nonexistent_invalid_arg(self, warsaw):
+    def test_tz_localize_nonexistent_invalid_arg(self):
         # GH 22644
-        tz = warsaw
+        tz = "Europe/Warsaw"
```

Comment:
why does this need to change?

Reply:
I didn't change that so I'm not sure, it changed when I merged with the main branch

Reply:
looks like something went wrong when merging, please try again

Reply:
It looks like there were some changes to these files on the pandas-dev main branch, I'll sync with my branch to get it up to date first

Reply:
Ok, so I merged with with the pandas/dev main branch, then re-added my changed with tests and pre-commit passing. I accidentally closed this while merging so if you can't re-open, the new pull request is here: https://github.com/pandas-dev/pandas/pull/50024